bindata/v4.1.0/kube-apiserver/pod.yaml

```diff
@@ -128,3 +130,6 @@ spec:
   - hostPath:
       path: /var/log/kube-apiserver
     name: audit-dir
+  - hostPath:
+      path: /etc/kubernetes/static-pod-resources/configmaps/trusted-ca-bundle-REVISION
```

Comment:
why this need to be revisioned?

Reply:
I don't think it has to be, it only felt consistent with the `kube-apiserver-pod` resource few lines above

Reply:
These files are in a revisioned directory. But due to the uniform mount point, that's not visible here.